### DISCUSSION TITLE
error

### DISCUSSION THREAD

**Discussion by xfixxy1356** [go-resty/resty#457] [General]
go get: github.com/go-resty/resty@v1.8.0 updating to
        github.com/go-resty/resty@v1.12.0: parsing go.mod:
        module declares its path as: gopkg.in/resty.v1
                but was required as: github.com/go-resty/resty

**Comment by MJjainam** [comment]
Facing the same issue.

**Comment by jeevatkm** [comment]
@MJjainam @xfixxy1356 The complete Resty v1 uses the `gopkg.in/resty.v1`.

Typically, the following `go.mod` definition should work.
```
require gopkg.in/resty.v1 v1.12.0
```

PS: 
It's been more than two years since the `v1.12.0` release, it is not actively supported. I would highly recommend you upgrade to v2. Have look at the Resty v2 release notes, it very simple/less breaking change it has - https://github.com/go-resty/resty/releases/tag/v2.0.0
